config/status_checks.yml

```diff
@@ -0,0 +1,88 @@
+deploys:
+  - app: 'identity-idp'
+    env: 'prod'
+    host: 'https://secure.login.gov'
+  - app: 'identity-sp-rails'
+    env: 'prod'
+  - app: 'identity-sp-sinatra'
+    env: 'prod'
+  - app: 'identity-dashboard'
+    env: 'prod'
+  - app: 'identity-openidconnect-sinatra'
+    env: 'prod'
```

Comment:
Configs without a host are not checked and shown as unavailable, so these are being "excluded" but we keep them in the UI for consistency (that's pre-existing behavior)